cluster-operator/src/main/java/io/strimzi/operator/cluster/model/CruiseControl.java

```diff
@@ -158,8 +165,7 @@ public class CruiseControl extends AbstractModel implements SupportsMetrics, Sup
      * Constructor
      *
      * @param reconciliation The reconciliation
-     * @param resource  Kubernetes resource with metadata containing the namespace and cluster name
-     * @param sharedEnvironmentProvider Shared environment provider
+     * @param resource       Kubernetes resource with metadata containing the namespace and cluster name
```

Comment:
Why was the `sharedEnvironmentProvider` removed from the Javadoc? That does not seem right.